docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -242,9 +238,22 @@ StackOverflow is also visited from time to time by the maintainers of MUI.
 
 [Post a question](https://stackoverflow.com/questions/tagged/mui)
 
-### Enterprise support
-
-We provide a [private support channel](https://support.mui.com/hc/en-us) for enterprise customers.
+### Professional support
+
+When purchasing an MUI X Pro, Premium license, depending on the plan, you get access to professional support for a limited duration.
+This support service is available on multiple channels.
+The recommended channel to open new issues is GitHub issues.
+If your support case has the need to share private information, you can also use Emails.
+
+- **MUI X Pro**: No SLA is provided but MUI's maintainers give these issues more attention than the ones from the Community plan. The channels:
+  - GitHub: [Open a new issue](https://github.com/mui-org/material-ui-x/issues/new/choose) and leave your Order ID.
+  - Emails (only to share private information): [Open a new issue](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email at x@mui.com.
+- **MUI X Premium**: Same as MUI X Pro with priority over Pro and a 48 hours SLA for the first answer.
+  - GitHub: this plan is not available yet
+  - Emails: this plan is not available yet
+- **MUI X Premium Priority**: Same as MUI X Premium but with a 24 hours SLA for the first answer and a pre-screening in under 4 hours.
```

Comment:
```suggestion
- **MUI X Premium Priority**: Same as MUI X Premium but with a 24 hours SLA for the first answer and a pre-screening in under 4 business hours.
```
? (In which case, what constitutes business hours?)

Reply:
I think that these 4 hours are going to be a real challenge to implement. It almost requires the team to be present on 3 overlapping timezones (8h x3). Plus, I doubt that it truly matters, AG Grid is one team in the UK and it seems to be good enough. I think that we can 

```suggestion
- **MUI X Premium Priority**: Same as MUI X Premium but with a 24 hours SLA for the first answer during working days.
```

Reply:
dry
```suggestion
- **MUI X Premium Priority**: Same as MUI X Premium but with a 24 hours SLA for the first answer.
```